client/deposits/details/test/index.js

```diff
@@ -35,18 +35,9 @@ describe( 'Deposit overview', () => {
 			isLoading: false,
 		} );
 
-		const overview = shallow(
+		const { container: overview } = render(
 			<DepositOverview depositId="po_mock" />
 		);
 		expect( overview ).toMatchSnapshot();
 	} );
 } );
-
-describe( 'Deposit details page', () => {
-	test( 'renders correctly', () => {
-		const page = shallow(
-			<DepositDetails query={ { id: 'po_mock' } } />
-		);
-		expect( page ).toMatchSnapshot();
-	} );
-} );
```

Comment:
I notice this tests disappeared entirely. Is there a reason it was deleted instead of rewritten or moved?

Reply:
Good question, I should have added a note on this somewhere. `<DepositDetails/>` component renders DepositOverview and TransactionsList without any logic. Each component is tested on its own. So I've removed it because it doesn't test anything new. 

Further refactoring probably make sense, i.e. Extracting Deposits overview and its tests into own files, but I thought it is out of scope of this PR.

Reply:
All right, sounds good!